homeassistant/components/assist_pipeline/websocket_api.py

```diff
@@ -1,16 +1,15 @@
 """Assist pipeline Websocket API."""
 
 import asyncio
-
-# Suppressing disable=deprecated-module is needed for Python 3.11
-import audioop  # pylint: disable=deprecated-module
 import base64
 from collections.abc import AsyncGenerator, Callable
 import contextlib
 import logging
 import math
 from typing import Any, Final
 
+# Suppressing disable=deprecated-module is needed for Python 3.11
+import audioop  # pylint: disable=deprecated-module
```

Comment:
```suggestion
import audioop  # pylint: disable=deprecated-module
```
Guess we could remove the comment here. `audioop` is resolved to the `audioop-lts` package now. 

_The `deprecated-module` check from pylint just looks at the import name, so that can't be removed. It might even be a slight positive to know that at some point we probably should replace this import for good._